pymc_marketing/mmm/base.py

```diff
@@ -1293,6 +1295,130 @@ def plot_channel_contribution_share_hdi(
     def graphviz(self, **kwargs):
         return pm.model_to_graphviz(self.model, **kwargs)
 
+    def _process_decomposition_components(self, data: pd.DataFrame):
+        """
+        Process data to compute the sum of contributions by component and calculate their percentages.
+        Parameters
+        ----------
+        dataframe : pd.DataFrame
+            Dataframe containing the time series data.
```

Comment:
Can you please more explicit about the columns and dimensions expected by this function? Think of it from a user perspective :)

Reply:
Added